site/docs/v1/tech/tutorials/integrate-flutter.md

```diff
@@ -0,0 +1,747 @@
+
+```shell
+cd $HOME/dev
+flutter create fusionauth_demo
+```
+
+After the installation process completes, you will see that the `fusionauth_demo` directory contains all the Flutter starter app configuration. Open the project directory with the text editor of your choice.
+
+You can run your new project in the actual device or an emulator to confirm everything is working before you customize any code. Do so by running the following command in the project directory:
+
+```shell
+cd $HOME/dev/fusionauth_demo
+flutter run
+```
+
+After running this, you will get the list of emulators or devices in which you want to run the project. You can simply choose the one in which you want the code to run. Make sure to remain in this directory for the rest of this tutorial.
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/flutter-choose-device.png" alt="Output of the flutter run command." class="img-fluid" figure=false %}
+
+However, to run the project on both iOS and Android together, you can use the following command:
+
+```shell
+flutter run -d all
+```
+
+The build process always takes a while the first time an app is built. After the successful build, you will get the boilerplate Flutter app in your emulators as displayed in the emulator screenshots below:
+
+{% include _image.liquid src="/assets/img/blogs/securing-flutter-app/flutter-demo-home-page.png" alt="The default Flutter app in the emulator." class="img-fluid" figure=false %}
+
+Now that you have a basic working application running, let's jump into the fun stuff: adding auth!
+
+## Integrating AppAuth
+
+[AppAuth](https://appauth.io/) is a popular OAuth package that can be used in both native and cross-platform mobile applications. In this project, you will be storing the access token using the secure storage package. Since such tokens allow your application to access protected resources such as APIs, you need to take care they are stored as securely as possible.
+
+Therefore, you need to install some dependencies to your project. For that, you need to open the `pubspec.yml` file in your project and add these dependencies:
+
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  http: ^0.12.1
+  flutter_appauth: ^0.9.1
+  flutter_secure_storage: ^3.3.3
+  simple_gravatar: ^1.0.5
+  cupertino_icons: ^1.0.0
+```
+
+## Setting up AppAuth
+
+Now that you have installed your libraries, you need to configure and add your previously configured callback URL to native configuration in both the Android and iOS directories.
+
+Let's look at Android first.
+
+### Android setup
+
+In your editor, you need to go to the `android/app/build.gradle` file for your Android app to specify the custom scheme. There should be a section in the file that looks similar to the below code block, but you'll need to add the FusionAuth URL: `com.fusionauth.flutterdemo`.
+
+At the end of your editing, make sure the `appAuthRedirectScheme` section looks similar to this:
+
+```gradle
+// ...
```

Comment:
Please import this using the `jekyll-remote-include` plugin from your sample app on github.